<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWindow</class>
 <widget class="QMainWindow" name="VideoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qconicalgradient(cx:1, cy:1, angle:16.7, stop:0 rgba(62, 53, 51, 255), stop:0.666667 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="playMeunwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>580</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:5px solid black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/icon/mclosepng.png);
border-radius: 40px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startPlayButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/icon/start.png);
border-radius: 40px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="playerListWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>221</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addVideoButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qconicalgradient(cx:1, cy:1, angle:16.7, stop:0 rgba(62, 53, 51, 255), stop:0.666667 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>播放列表</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
